docs/settings/logging-settings.asciidoc

```diff
@@ -4,6 +4,16 @@
 <titleabbrev>Logging settings</titleabbrev>
 ++++
 
+{kib} relies on three high-level entities to set the logging service: appenders, loggers, and root.
+
+- Appenders define where log messages are displayed (stdout or console) and stored (file on the disk), and their layout.
```

Comment:
Logs aren't stored by default, they're only stored if a `file` type appender is configured for them. Users might interpret this sentence to mean they are stored by default, it's just where they're stored that they have control over. I'm not sure how to improve this though, maybe:
```suggestion
- Appenders define where log messages are displayed (stdout or console) and their layout (pattern or json). They also allow you to specify if you want the logs stored and, if so, where (file on the disk).
```